src/feature/NodeGraphFeature.ts

```diff
@@ -126,7 +127,7 @@ class NodeGraphWebViewProvider implements vscode.Disposable {
       graphContent = graphContent.replace(/\\/g,"\\\\");
       graphContent = graphContent.replace(`label = "editorservices"`,styling);
 
-      svgContent = viz(graphContent,"svg");
+      svgContent = new Viz({Module, render}).renderString(graphContent);
```

Comment:
`renderString` returns a promise, not a string. So it's not working.

e.g.

![image](https://user-images.githubusercontent.com/5819622/72593975-4d413d00-3941-11ea-87a8-a55ae5997459.png)


Will need to rework 

https://github.com/lingua-pupuli/puppet-vscode/blob/68d21969d13a19f8d7ce9287c751ecafa53206d6/src/feature/NodeGraphFeature.ts#L94

and

https://github.com/lingua-pupuli/puppet-vscode/blob/68d21969d13a19f8d7ce9287c751ecafa53206d6/src/feature/NodeGraphFeature.ts#L103

To pass the promise through and resolve it.